src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -1335,6 +1335,96 @@ public async Task ServerThrowsHubExceptionIfBuildingAsyncEnumeratorIsNotPossible
             }
         }
 
+        [Theory]
+        [QuarantinedTest]
```

Comment:
Why are the tests pre-quarantined?

Reply:
There was an idea floating around (cc @JamesNK) that we consider introducing tests as pre-quarantined. I'm not 100% sold on doing it for all tests, but in areas that have possible timing/races, I think it's something to consider.

I'm fine either way with this one, plenty of other tests have this pattern and have not flaked out.